src/main/java/ResponseHandler.java

```diff
@@ -0,0 +1,10 @@
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class ResponseHandler {
+
+    public void prepare(OutputStream output, String statusLine) throws IOException {
```

Comment:
I find it odd that a method to prepare some output actually writes and flushes the output. Could this be named differently?